src/apps/service/util.clj

```diff
@@ -1,18 +1,38 @@
 (ns apps.service.util
-  (:use [apps.transformers :only [param->long]]
+  (:use [apps.routes.schemas.app :only [AdminAppListingJobStatsKeys]]
+        [apps.transformers :only [param->long]]
         [apps.util.conversions :only [remove-nil-vals]])
   (:require [clojure.string :as string]
             [clojure-commons.exception-util :as cxu]
             [kameleon.uuids :as uuids])
   (:import [java.util UUID]))
 
+(defn- app-sorter-keyfn
+  [sort-field]
+  (let [sort-field (keyword sort-field)]
+    (condp contains? sort-field
+      AdminAppListingJobStatsKeys (comp sort-field :job_stats)
+      #{:average :total}          (comp sort-field :rating)
```

Comment:
I don't think it's possible to display descriptions for these enums in the swagger docs, since these values are only displayed in the drop-down list for the query param.